crane/data.py

```diff
@@ -106,6 +115,9 @@ def load_all(app):
     :param app: the flask application
     :type  app: flask.Flask
     """
+    global response_data_v2
+    repos_v2 = {}
+
     global response_data
```

Comment:
Renaming reponse_data to v1_response_data and response_data_v2 to v2_response_data would be good for consistency.